tests/ChatTests.hs

```diff
@@ -4249,7 +4299,7 @@ itemId :: Int -> String
 itemId i = show $ length chatFeatures + i
 
 groupItemId :: Int -> Int -> String
-groupItemId n i = show $ (length chatFeatures) * n + i
+groupItemId n i = show $ length chatFeatures * n + i
```

Comment:
why does it use `chatFeatures` and not `groupFeatures`?
```suggestion
groupItemId n i = show $ length groupFeatures * n + i
```

Reply:
ah - that's why the test was failing... :)

Reply:
check out the change in my direct messages commit - I just changed numbers there incorrectly then.